index.html

```diff
@@ -1,14 +1,459 @@
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_02.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                GDG content
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Google technologies in areas like Android, Flutter and so much
+                more!
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_03.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                GDG Forum
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Share your thoughts and opinions with experts for each topic.
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_04.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                Workshops
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Full-day workshops, hands-on experiences, and debugging events.
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_05.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                Networking
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Get opportunities to network with Coders around the world.
+              </p>

```

Comment:
- [ ] Kindly make the items in each row align with their corresponding item in other rows in the _**Main Program section**_. Currently, the items in a card are not aligned on the same horizontal lines with their counter paths in other cards in the section. Kindly fix this. 👍 💻 


### Your Design

![image](https://user-images.githubusercontent.com/77106962/225111465-c973732c-62eb-4150-b21c-4f63051365b1.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/225111925-bf10c48f-f15b-4d44-9272-f610553f2fe1.png)